sssom/util.py

```diff
@@ -85,22 +89,23 @@ class MappingSetDataFrame:
     metadata: Optional[MetadataType] = None  # header metadata excluding prefixes
 
     def merge(
-        self, msdf2: "MappingSetDataFrame", inplace: bool = True
+        self, *msdfs: "MappingSetDataFrame", inplace: bool = True
```

Comment:
Why is MappingSetDataFrame in quotes? Just for my curiosity :)

Reply:
I think when a reference of a class is made within the same class, it should be in quotes, else it throws an error.

Reply:
I think when a reference of a class is made within the same class, it should be in quotes, else it throws an error.

Reply:
Cool thanks!